src/routes/transactions/mappers/common/native-staking.mapper.spec.ts

```diff
@@ -406,6 +406,7 @@ describe('NativeStakingMapper', () => {
 
       const actual = await target.mapValidatorsExitInfo({
         chainId: chain.chainId,
+        safeAddress: transaction.safe,
```

Comment:
On face value it looks as though we could access the `safe` property in this method but `transaction` can be `null` (when proposing a transaction). We therefore need explicitly pass the `safeAddress`.